great_expectations/expectations/registry.py

```diff
@@ -65,9 +65,36 @@ def register_renderer(
 
 
 def get_renderer_names(object_name: str) -> List[str]:
+    """Gets renderer names for a given Expectation or Metric.
+
+    Args:
+        object_name: The name of an Expectation or Metric for which to get renderer names.
+
+    Returns:
+        A list of renderer names for the Expectation or Metric.
+    """
```

Comment:
Could we modify our internal linters (docstrings/type hints)?

Reply:
I'm not following what the question is here. I documented this to keep it in line with the method I added (`get_renderer_names_from_renderer_prefix`).

Reply:
ah sorry I meant the thresholds in `scripts/`. We have constants in `check_type_hint_coverage.py` and `check_docstring_coverage.py` that need updating 😄